### ISSUE TITLE
[bitnami/discourse] Ruby 2.7.0 being packaged up as 2.7.6 in Discourse image

### ISSUE THREAD

**Issue by lju-lazarevic** [bitnami/containers#12936]
### Name and Version

bitnami/discourse:2.8.8+

### What steps will reproduce the bug?

- Based on this dockerfile: https://github.com/bitnami/containers/blob/main/bitnami/discourse/2/debian-11/Dockerfile
- Downloaded https://downloads.bitnami.com/files/stacksmith/ruby-2.7.6-154-linux-amd64-debian-11.tar.gz 
- Unpacked and navigated to lib
- Can see that it's version 2.7.0 of Ruby being used, and not 2.7.6

### What is the expected behavior?

Ruby 2.7.6 installed, thereby avoiding some critical CVEs

### What do you see instead?

Ruby 2.7.0 installed, along with CVEs (when scanned with Trivy)

**Comment by javsalgar** [comment]
Hi,

Please, note this asset is a community-supported solution. This means that the Bitnami team is not actively working on new features/improvements nor providing support through GitHub Issues. Any new issue will stay open for 20 days to allow the community to contribute, after 15 days without activity the issue will be marked as stale being closed after 5 days.

The Bitnami team will review any PR that is created, feel free to create a PR if you find any issue or want to implement a new feature.

New versions and releases cadence are not going to be affected. Once a new version is released in the upstream project, the Bitnami container image will be updated to use the latest version, supporting the different branches supported by the upstream project as usual.

Having said that, the container has the correct installed Ruby

```
discourse 16:29:21.73 Welcome to the Bitnami discourse container
discourse 16:29:21.73 Subscribe to project updates by watching https://github.com/bitnami/containers
discourse 16:29:21.73 Submit issues and feature requests at https://github.com/bitnami/containers/issues
discourse 16:29:21.73

root@abd8690ab601:/#
root@abd8690ab601:/#
root@abd8690ab601:/# ruby --version
ruby 2.7.6p219 (2022-04-12 revision c9c2245c0a) [x86_64-linux]
```

**Comment by lju-lazarevic** [comment]
Hi Javier,

Thanks for the message. I will double-check again, when when it was installed, we can see that ruby 2.7.6 was installed, but vulnerabilities that are coming up are ones that were fixed in 2.7.6 are coming up. Changing the dockerfile and manually installing 2.7.6 gets rid of one of the critical CVEs. The other two seem to be coming from gems. I will try and dig about to see what's happening re the gems, but not really sure what to suggest why 2.7.0 seems to be being used for the 3rd CVE.

Kind regards,

Lju
